### PR TITLE
Settings

### PR DESCRIPTION
**Pull request by joelevin**
Settings coordinator

### CODE REVIEW

File: weTranslate/SettingsCoordinator.swift [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+final class SettingsCoordinator: CoordinatorType {
+    
+    // MARK: - Properties
+    let navigationController: UINavigationController
+    var childCoordinators: [CoordinatorType] = []
+    
+    // MARK: - Initialization
+    init(navigationController: UINavigationController)
+    {
```
Comment (jenniferlim) [inline weTranslate/SettingsCoordinator.swift:19]: Could you pls update to curly brackets ?

File: weTranslate/SettingsViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+//  Copyright © 2016 weTranslate. All rights reserved.
+//
+
+import Foundation
+
+struct SettingsViewModel {
+    // MARK: - Properties
+    
+    // MARK: - Methods
+}
```
Comment (jenniferlim) [inline weTranslate/SettingsViewModel.swift:15]: Could you add a newline at the EOF?

**Comment by jenniferlim** [comment]
:bowtie:

File: weTranslate/SettingsViewController.swift [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+//
+
+import UIKit
+
+class SettingsViewController: UIViewController {
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
```
Comment (lionel-alves) [inline weTranslate/SettingsViewController.swift:16]: Remove comment.

File: weTranslate/SettingsViewController.swift [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
```
Comment (lionel-alves) [inline weTranslate/SettingsViewController.swift:22]: Remove `didReceiveMemoryWarning` method.

File: weTranslate/SettingsViewController.swift [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
```
Comment (lionel-alves) [inline weTranslate/SettingsViewController.swift:23]: :scissors:

File: weTranslate/SettingsCoordinator.swift [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+//
+//  Created by Joe Levin on 2/27/16.
+//  Copyright © 2016 weTranslate. All rights reserved.
+//
+
+import UIKit
+
+final class SettingsCoordinator: CoordinatorType {
+    
+    // MARK: - Properties
```
Comment (lionel-alves) [inline weTranslate/SettingsCoordinator.swift:13]: Newline.

File: weTranslate/SettingsCoordinator.swift [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+    let navigationController: UINavigationController
+    var childCoordinators: [CoordinatorType] = []
+
```
Comment (lionel-alves) [inline weTranslate/SettingsCoordinator.swift:16]: Newline. (two newlines before marks except for the first one)

File: weTranslate/SettingsCoordinator.swift [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+import UIKit
+
+final class SettingsCoordinator: CoordinatorType {
+    
+    // MARK: - Properties
+    let navigationController: UINavigationController
+    var childCoordinators: [CoordinatorType] = []
+    
+    // MARK: - Initialization
```
Comment (lionel-alves) [inline weTranslate/SettingsCoordinator.swift:17]: Newline.

File: weTranslate/SettingsCoordinator.swift [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+    init(navigationController: UINavigationController)
+    {
+        self.navigationController = navigationController;
```
Comment (lionel-alves) [inline weTranslate/SettingsCoordinator.swift:20]: `;` :scream:

File: weTranslate/SettingsViewModel.swift [hunk 10]
```diff
@@ -0,0 +1,15 @@
...
+//  SettingsViewModel.swift
+//  weTranslate
+//
+//  Created by Joe Levin on 2/27/16.
+//  Copyright © 2016 weTranslate. All rights reserved.
+//
+
+import Foundation
+
+struct SettingsViewModel {
```
Comment (lionel-alves) [inline weTranslate/SettingsViewModel.swift:11]: Newline.

File: weTranslate/SettingsViewModel.swift [hunk 11]
```diff
@@ -0,0 +1,15 @@
...
+    // MARK: - Properties
+
```
Comment (lionel-alves) [inline weTranslate/SettingsViewModel.swift:13]: Newline.

File: weTranslate/SettingsViewModel.swift [hunk 12]
```diff
@@ -0,0 +1,15 @@
...
+//  Created by Joe Levin on 2/27/16.
+//  Copyright © 2016 weTranslate. All rights reserved.
+//
+
+import Foundation
+
+struct SettingsViewModel {
+    // MARK: - Properties
+    
+    // MARK: - Methods
```
Comment (lionel-alves) [inline weTranslate/SettingsViewModel.swift:14]: Newline.

File: weTranslate/SettingsCoordinator.swift (hunk 1 above)
Comment (lionel-alves) [inline weTranslate/SettingsCoordinator.swift:19]: +1

**Comment by lionel-alves** [comment]
A bunch of nits, need a rebase, otherwise :bowtie:.

File: weTranslate/SettingsCoordinator.swift (hunk 9 above)
Comment (joelevin) [inline weTranslate/SettingsCoordinator.swift:20]: 8)

**Comment by lionel-alves** [comment]
@joelevin do you want to fix the comments so we can merge this?

**Comment by lionel-alves** [comment]
Closing this PR, feel free to open a new one if you have time to work on it.

**Pull request closed without merging**